index.js

```diff
@@ -1,88 +1,125 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+var counter = 1
+var counter = 2
```

Comment:
This does work, but the idea here is to get away from using `var` and instead use `let` so we're only reassigning the variables we mean to! So you could instead do

`let counter = 1`
`counter = 2`

This isn't a big deal in such a small example, but better to get into the habit now!